examples/using_blocks.py

```diff
@@ -0,0 +1,52 @@
+import os
```

Comment:
We already use blocks on some of the other examples, i.e: `actions.py`.
I'm not sure if it's a great idea to dedicate a whole example just to this, perhaps we could add it to one of the existent files?
How was your experience as a user? Do you feel this will be helpful?

Reply:
If I teleported back to the first time looking at the existing examples, yes. But that's just my opinion, up to you if you want it changed. Let me know